hack/ci-e2e-kind-upgrade.sh

```diff
@@ -183,7 +177,7 @@ kind_up
 trap "
 ( rm -rf $GARDENER_RELEASE_DOWNLOAD_PATH/gardener-releases);
 ( export_logs '$CLUSTER_NAME'; export_events_for_kind '$CLUSTER_NAME'; export_events_for_shoots )
-( kind_down;)
```

Comment:
Is this left in debugging? I understand that you want to keep things running to debug what went wrong, but do we really want to change this default?

Reply:
Yes, I added for debugging purpose.
Now reverted it.